code/src/components/Tech.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+import { ContainerWrapper, FlexWrapper, SectionWrapper, SectionHeadings, TextSection } from 'GlobalStyles';
+
+// react component
+
+export const Tech = () => {
+  return (
+    <SectionWrapper bgColor="#f8e9f0">
+      <ContainerWrapper>
+        <FlexWrapper>
+          <SectionHeadings>Tech</SectionHeadings>
+          <TextSection>
+            <SpanBold> CSS3 </SpanBold>
```

Comment:
Here I see an example of how you used global styles in a very good way. Nice job, I will try to do it like this to. :)